crates/db_schema/src/lib.rs

```diff
@@ -75,6 +76,7 @@ pub enum SortType {
   TopSixMonths,
   TopNineMonths,
   Controversial,
+  Scaled,
 }
```

Comment:
Please write the docs before merging this, otherwise we will forget it.

Reply:
Done